From 0a1f52e469679ba51cbd48124951174fd4dbe01a Mon Sep 17 00:00:00 2001
From: Ben Greear <greearb@candelatech.com>
Date: Fri, 21 May 2021 16:11:01 -0700
Subject: [PATCH 09/15] intel-oot:  Rename modules so that we can compile both
 drivers.

This lets me compile the oot drivers as well as the in-kernel
drivers.  There are some warnings about double exports, and probably
some tricks will need to be played at install time to use one or
the other.

Signed-off-by: Ben Greear <greearb@candelatech.com>
---
 .../net/wireless/intel-oot/iwlwifi/Makefile   | 56 +++++++++----------
 .../wireless/intel-oot/iwlwifi/mvm/Makefile   | 26 ++++-----
 2 files changed, 41 insertions(+), 41 deletions(-)

diff --git a/drivers/net/wireless/intel-oot/iwlwifi/Makefile b/drivers/net/wireless/intel-oot/iwlwifi/Makefile
index e71985ed00770..e197b67963819 100644
--- a/drivers/net/wireless/intel-oot/iwlwifi/Makefile
+++ b/drivers/net/wireless/intel-oot/iwlwifi/Makefile
@@ -1,39 +1,39 @@
 # SPDX-License-Identifier: GPL-2.0
 # common
-obj-$(CONFIG_OOT_IWLWIFI)	+= iwlwifi.o
-iwlwifi-objs		+= iwl-io.o
-iwlwifi-objs		+= iwl-drv.o
-iwlwifi-objs		+= iwl-debug.o
-iwlwifi-objs		+= iwl-eeprom-read.o iwl-eeprom-parse.o
-iwlwifi-objs		+= iwl-phy-db.o iwl-nvm-parse.o
-iwlwifi-objs		+= iwl-dbg-tlv.o
-iwlwifi-objs		+= iwl-trans.o
-iwlwifi-objs		+= queue/tx.o
-
-iwlwifi-objs		+= fw/img.o fw/notif-wait.o
-iwlwifi-objs		+= fw/dbg.o fw/pnvm.o fw/dump.o
-iwlwifi-$(CONFIG_OOT_IWLMVM)	+= fw/paging.o fw/smem.o fw/init.o
-iwlwifi-$(CONFIG_OOT_IWLXVT)	+= fw/init.o
-iwlwifi-$(CONFIG_OOT_IWLWIFI_DEBUGFS) += fw/debugfs.o
-
-iwlwifi-$(CONFIG_ACPI)	+= fw/acpi.o
-iwlwifi-$(CONFIG_EFI)	+= fw/uefi.o
+obj-$(CONFIG_OOT_IWLWIFI)	+= iwlwifi-oot.o
+iwlwifi-oot-objs		+= iwl-io.o
+iwlwifi-oot-objs		+= iwl-drv.o
+iwlwifi-oot-objs		+= iwl-debug.o
+iwlwifi-oot-objs		+= iwl-eeprom-read.o iwl-eeprom-parse.o
+iwlwifi-oot-objs		+= iwl-phy-db.o iwl-nvm-parse.o
+iwlwifi-oot-objs		+= iwl-dbg-tlv.o
+iwlwifi-oot-objs		+= iwl-trans.o
+iwlwifi-oot-objs		+= queue/tx.o
+
+iwlwifi-oot-objs		+= fw/img.o fw/notif-wait.o
+iwlwifi-oot-objs		+= fw/dbg.o fw/pnvm.o fw/dump.o
+iwlwifi-oot-$(CONFIG_OOT_IWLMVM)	+= fw/paging.o fw/smem.o fw/init.o
+iwlwifi-oot-$(CONFIG_OOT_IWLXVT)	+= fw/init.o
+iwlwifi-oot-$(CONFIG_OOT_IWLWIFI_DEBUGFS) += fw/debugfs.o
+
+iwlwifi-oot-$(CONFIG_ACPI)	+= fw/acpi.o
+iwlwifi-oot-$(CONFIG_EFI)	+= fw/uefi.o

 # Bus
-iwlwifi-$(CONFIG_PCI) += pcie/drv.o pcie/rx.o pcie/tx.o pcie/trans.o
-iwlwifi-$(CONFIG_PCI) += pcie/ctxt-info.o pcie/ctxt-info-gen3.o pcie/trans-gen2.o pcie/tx-gen2.o
+iwlwifi-oot-$(CONFIG_PCI) += pcie/drv.o pcie/rx.o pcie/tx.o pcie/trans.o
+iwlwifi-oot-$(CONFIG_PCI) += pcie/ctxt-info.o pcie/ctxt-info-gen3.o pcie/trans-gen2.o pcie/tx-gen2.o

-iwlwifi-$(CONFIG_OOT_IWLMVM) += cfg/7000.o cfg/8000.o
-iwlwifi-$(CONFIG_OOT_IWLMVM) += cfg/9000.o cfg/22000.o
+iwlwifi-oot-$(CONFIG_OOT_IWLMVM) += cfg/7000.o cfg/8000.o
+iwlwifi-oot-$(CONFIG_OOT_IWLMVM) += cfg/9000.o cfg/22000.o

-iwlwifi-$(CONFIG_OOT_IWLWIFI_SUPPORT_DEBUG_OVERRIDES) += iwl-dbg-cfg.o
+iwlwifi-oot-$(CONFIG_OOT_IWLWIFI_SUPPORT_DEBUG_OVERRIDES) += iwl-dbg-cfg.o

-iwlwifi-objs += $(iwlwifi-m)
-iwlwifi-objs += $(iwlwifi-y)
+iwlwifi-oot-objs += $(iwlwifi-oot-m)
+iwlwifi-oot-objs += $(iwlwifi-oot-y)

-iwlwifi-$(CONFIG_OOT_IWLWIFI_DEVICE_TRACING) += iwl-devtrace.o
-iwlwifi-$(CONFIG_OOT_IWLWIFI_DEVICE_TESTMODE) += iwl-tm-gnl.o iwl-dnt-cfg.o iwl-dnt-dispatch.o
-iwlwifi-$(CONFIG_OOT_IWLWIFI_DEVICE_TESTMODE) += iwl-dnt-dev-if.o fw/testmode.o
+iwlwifi-oot-$(CONFIG_OOT_IWLWIFI_DEVICE_TRACING) += iwl-devtrace.o
+iwlwifi-oot-$(CONFIG_OOT_IWLWIFI_DEVICE_TESTMODE) += iwl-tm-gnl.o iwl-dnt-cfg.o iwl-dnt-dispatch.o
+iwlwifi-oot-$(CONFIG_OOT_IWLWIFI_DEVICE_TESTMODE) += iwl-dnt-dev-if.o fw/testmode.o

 ccflags-y += -I$(src)
 subdir-ccflags-y += $(call cc-option,-Wimplicit-fallthrough)
diff --git a/drivers/net/wireless/intel-oot/iwlwifi/mvm/Makefile b/drivers/net/wireless/intel-oot/iwlwifi/mvm/Makefile
index a97313fefec5f..1af66fd30cb53 100644
--- a/drivers/net/wireless/intel-oot/iwlwifi/mvm/Makefile
+++ b/drivers/net/wireless/intel-oot/iwlwifi/mvm/Makefile
@@ -1,20 +1,20 @@
 # SPDX-License-Identifier: GPL-2.0
-obj-$(CONFIG_OOT_IWLMVM)   += iwlmvm.o
-iwlmvm-y += fw.o mac80211.o nvm.o ops.o phy-ctxt.o mac-ctxt.o
-iwlmvm-y += utils.o rx.o rxmq.o tx.o binding.o quota.o sta.o sf.o
-iwlmvm-y += scan.o time-event.o rs.o rs-fw.o
-iwlmvm-y += power.o coex.o
-iwlmvm-y += tt.o offloading.o tdls.o
-iwlmvm-y += ftm-responder.o ftm-initiator.o
-iwlmvm-y += rfi.o
-iwlmvm-y += nan.o
-iwlmvm-$(CONFIG_OOT_IWLWIFI_DEBUGFS) += debugfs.o debugfs-vif.o
-iwlmvm-$(CONFIG_OOT_IWLWIFI_LEDS) += led.o
-iwlmvm-$(CONFIG_PM) += d3.o
+obj-$(CONFIG_OOT_IWLMVM)   += iwlmvm-oot.o
+iwlmvm-oot-y += fw.o mac80211.o nvm.o ops.o phy-ctxt.o mac-ctxt.o
+iwlmvm-oot-y += utils.o rx.o rxmq.o tx.o binding.o quota.o sta.o sf.o
+iwlmvm-oot-y += scan.o time-event.o rs.o rs-fw.o
+iwlmvm-oot-y += power.o coex.o
+iwlmvm-oot-y += tt.o offloading.o tdls.o
+iwlmvm-oot-y += ftm-responder.o ftm-initiator.o
+iwlmvm-oot-y += rfi.o
+iwlmvm-oot-y += nan.o
+iwlmvm-oot-$(CONFIG_OOT_IWLWIFI_DEBUGFS) += debugfs.o debugfs-vif.o
+iwlmvm-oot-$(CONFIG_OOT_IWLWIFI_LEDS) += led.o
+iwlmvm-oot-$(CONFIG_PM) += d3.o

 ccflags-y += -I$(src)/../

 # non-upstream things
-iwlmvm-$(CONFIG_OOT_IWLMVM_VENDOR_CMDS) += vendor-cmd.o
+iwlmvm-oot-$(CONFIG_OOT_IWLMVM_VENDOR_CMDS) += vendor-cmd.o
 ifeq ($(CONFIG_OOT_IWLMVM_VENDOR_CMDS),y)
 endif
--
2.26.3
